velox/docs/functions/binary.rst

```diff
@@ -34,6 +34,14 @@ Binary Functions
 
     Computes the HMAC with sha512 of ``binary`` with the given ``key``.
 
+.. function:: spooky_hash_v2_32(binary) -> varbinary
+
+    Computes the SpookyHashV2 32-bit hash of ``binary``.
+
+.. function:: spooky_hash_v2_64(binary) -> varbinary
```

Comment:
We usually maintain the function names in alphabetical order in the files. But it doesn't seem like that is followed in this file.

Please re-order the functions (though in a separate PR).

Reply:
Yes, I will open a separate PR to fix the order in the file, thanks for the comment!